vitest.config.ts

```diff
@@ -1,28 +1,21 @@
 import { defineConfig } from "vitest/config";
 
-const isPostgres = process.env.DIALECT === "postgresql";
+const service = process.env.SERVICE;
```

Comment:
service means what?

http service? db service? dbType may be bit more clear if that's the only point of this.

Reply:
Replaced `SERVICE` which was referring to the service name (database+version) in the docker compose file to `REPORTS_DIRECTORY`